rasa/core/policies/ted_policy.py

```diff
@@ -895,15 +896,37 @@ def _check_data(self) -> None:
 
     def _prepare_layers(self) -> None:
         for name in self.data_signature.keys():
-            self._prepare_sparse_dense_layer_for(name, self.data_signature)
             if name in SEQUENCE_FEATURES_TO_ENCODE:
-                self._prepare_sequence_layers(name)
+                self._tf_layers[
+                    f"{name}_sequence_layer"
+                ] = rasa_layers.RasaSequenceLayer(
+                    name, self.data_signature[name], self.config
+                )
+            elif SENTENCE in self.data_signature[name]:
```

Comment:
do we need this check? shall we just use `else`?

Reply:
When I didn't use it, it failed on `dialogue` features I think.

Reply:
aaah, makes sense